In [ ]:

You work for an environmental think tank called Repair Our Air (ROA). ROA is formulating policy recommendations to improve the air quality in America, using the Environmental Protection Agency's Air Quality Index (AQI) to guide their decision making. An AQI value close to 0 signals "little to no" public health concern, while higher values are associated with increased risk to public health.

They've tasked you with leveraging AQI data to help them prioritize their strategy for improving air quality in America.

ROA is considering the following decisions. For each, construct a hypothesis test and an accompanying visualization, using your results of that test to make a recommendation:

ROA is considering a metropolitan-focused approach. Within California, they want to know if the mean AQI in Los Angeles County is statistically different from the rest of California.
With limited resources, ROA has to choose between New York and Ohio for their next regional office. Does New York have a lower AQI than Ohio?
A new policy will affect those states with a mean AQI of 10 or greater. Would Michigan be affected by this new policy?
Notes:

For your analysis, you'll default to a 5% level of significance.

In [1]:
import pandas as pd
from scipy import stats

In [2]:
df=df=pd.read_csv('c4_epa_air_quality.csv')

This is an air quality index dataset Source-Google Advanced Data Analytics Certification Modified Dataset

In [5]:
df.head()

Unnamed: 0  date_local    state_name   county_name      city_name  \
0           0  2018-01-01       Arizona      Maricopa        Buckeye   
1           1  2018-01-01          Ohio       Belmont      Shadyside   
2           2  2018-01-01       Wyoming         Teton  Not in a city   
3           3  2018-01-01  Pennsylvania  Philadelphia   Philadelphia   
4           4  2018-01-01          Iowa          Polk     Des Moines   

                                     local_site_name   parameter_name  \
0                                            BUCKEYE  Carbon monoxide   
1                                          Shadyside  Carbon monoxide   
2  Yellowstone National Park - Old Faithful Snow ...  Carbon monoxide   
3                             North East Waste (NEW)  Carbon monoxide   
4                                          CARPENTER  Carbon monoxide   

    units_of_measure  arithmetic_mean  aqi  
0  Parts per million         0.473684    7  
1  Parts per million         0.263158    5  
2  Parts per million         0.111111    2  
3  Parts per million         0.300000    3  
4  Parts per million         0.215789    3

ROA is considering a metropolitan-focused approach. Within California, they want to know if the mean AQI in Los Angeles County is statistically different from the rest of California.


Comparing sample means of 2 independent samples we use two sample t test

In [21]:
Los_Angeles_County_AQI=df[df.county_name=='Los Angeles'].aqi

In [22]:
Rest_of_California_AQI=df[(df.state_name=='California') & (df.county_name!='Los Angeles')].aqi

In [24]:
comparing_aqi=stats.ttest_ind(a=Los_Angeles_County_AQI,b=Rest_of_California_AQI,equal_var=False)

In [25]:
comparing_aqi

Ttest_indResult(statistic=2.1107010796372014, pvalue=0.049839056842410995)

Null Hypothesis=Within California,mean AQI in Los Angeles County is not statistically different from the rest of California.

Alternate Hypothesis=Within California, the mean AQI in Los Angeles County is statistically different from the rest of California.

0.049 or 4.9% is less than 5% so null hypothesis is rejected

With limited resources, ROA has to choose between New York and Ohio for their next regional office. Does New York have a lower AQI than Ohio?


Camparing sample means of two independent samples in one direction using two sample t test

In [30]:
Ny_Oh_aqi=stats.ttest_ind(a=df[df.state_name=='New York'].aqi,b=df[df.state_name=='Ohio'].aqi,alternative='less',equal_var=False)

In [31]:
Ny_Oh_aqi

Ttest_indResult(statistic=-2.025951038880333, pvalue=0.030446502691934697)

Null Hypothesis:New York and Ohion have same AQI
Alternate Hypothesis:New York has lower AQI than Ohio
0.030 or 3% is less than significance level of 5% so null hypothesis rejected
The regional office can be setup in  New York 

A new policy will affect those states with a mean AQI of 10 or greater. Will Michigan be affected by this new policy?¶

Here we are comparing a mean sample with a relative value in one direction so we use one sample t test

In [37]:
Mic_aqi=stats.ttest_1samp(df[df.state_name=='Michigan'].aqi,10,alternative='greater')

In [38]:
Mic_aqi

Ttest_1sampResult(statistic=-1.7395913343286131, pvalue=0.9399405193140109)

With a p-value (0.940) being greater than 0.05 (as your significance level is 5%)  fail to reject the null hypothesis.

Therefore, you cannot conclude at the 5% significance level that Michigan's mean AQI is greater than 10. This implies that Michigan would most likely not be affected by the new policy.